azurerm/resource_arm_app_service.go

```diff
@@ -239,6 +248,23 @@ func resourceArmAppService() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+
+			"source_control": {
+				Type:     schema.TypeList,
+				Optional: true,
```

Comment:
can we remove this `Optional` assignment? it's only needed for user specifiable fields (and not readonly ones)

Reply:
Good catch.